main.go

```diff
@@ -562,5 +564,9 @@ func timeoutMs(timeout int) time.Duration {
 }
 
 func printVersion(_ *cobra.Command, _ []string) {
-	fmt.Println("Authorino", version)
+	if dirty == "true" {
+		fmt.Printf("Authorino %s (%s-dirty)\n", version, gitSHA)
```

Comment:
Am I wrong or there was a suggestion for printing this as JSON?

```json
{"version":"latest","commit":"abcd1234","dirty":true}
```

Was there any Kuadrant-wide decision about the format?

Reply:
I don't believe so... 

I know for that format, that is the one we use in the logs. Example:
```bash
{"level":"info","ts":"2024-08-26T10:24:20+01:00","logger":"authorino","msg":"build information","version":"471-add-commit-hash-implementation-similar-to-kuadrantctl-to-authorino","commit":"2164661fe2f879fe07b5a1fa11ece02c75e8cef2","dirty":"false","cmd":"server"}
```

As for when we use `authorino version` , that one is not JSON, and I don't believe there was a suggestion for doing this just yet.

```bash
evanhearne@fedora:~/Documents/projects/authorino$ ./bin/authorino version
Authorino 471-add-commit-hash-implementation-similar-to-kuadrantctl-to-authorino (2164661fe2f879fe07b5a1fa11ece02c75e8cef2)
```

Sometime in the future, I believe there will be a PR opened to suggest how the version command will look like universally across all operators.

Reply:
No problem. I probably got that wrong then. It works for me the way it is now.